docs/data/sources/airship.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Import Airship data
+description: Adding Airship’s unique information to Amplitude gives you a full view of what’s going on in and around your app.
+---
+
+Airship Real-Time Data Streaming (RTDS) delivers user-level information about push sends, conversions, and uninstall events. Adding Airship’s unique information to Amplitude gives you a full view of what’s going on in and around your app.
+
+!!!note "Other Amplitude + Airship integrations"
+
+    This integration imports Airship data into Amplitude. Amplitude offers one other integrations with AppsFlyer: 
+
+    - [Send cohorts to AppsFlyer](/data/destinations/airship-cohort)
+
+## About Airship
+
+[Airship](https://www.airship.com/) provides mobile advertising attribution and analytics, helping marketers to pinpoint their targeting optimize their ad spend and boost their ROI. Send your AppsFlyer data to Amplitude to see how your different acquisition sources are performing.
```

Comment:
Is this accurate to Airship or is it AppsFlyer's description?

Reply:
Whoops! Good catch Cynthia. I updated the blurb changes.